### DISCUSSION TITLE
Integrating a model with Avalanche

### DISCUSSION THREAD

**Discussion by YaegerKnight** [ContinualAI/avalanche#1205] [Q&A] [answered]
Hello,

I have a neural network I trained for speech recognition. Does anyone know if it is possible to integrate models with avalanche ? I tried looking at the documentation but I was not able to find any code examples for the same. 

Thank you.

**Comment by vlomonaco** [comment, marked as answer]
Hi @YaegerKnight! Thanks for your interest in Avalanche!
Have you checked these resources?

- https://avalanche.continualai.org/from-zero-to-hero-tutorial/02_models
- https://avalanche.continualai.org/examples/models

In a nutshell we support any PyTorch model and we give a little extra support for models that change over time or needs to support multiple "heads": https://avalanche-api.continualai.org/en/v0.2.1/models.html
